.github/workflows/build_rust.yaml

```diff
@@ -74,6 +74,22 @@ jobs:
       - name: Copy binary to release directory POSIX
         if: matrix.os != 'windows-latest'
         run: cp target/release/idf-im-cli release/${{ matrix.os }}/eim
+      
+      - name: Codesign macOS build artifacts
+        if: matrix.os == 'macos-latest'
```

Comment:
please change the condition to `startsWith(matrix.os, 'macos')` as this way we do not sign build for intel based macs

Reply:
thanks for the review, addressed it.